src/EventEdition/InfoPanel.vala

```diff
@@ -185,6 +208,17 @@ public class Maya.View.EventEdition.InfoPanel : Gtk.Grid {
         attach (frame, 0, 9, 5, 1);
 
         load ();
+
+        calchooser.bind_property ("current-source", current_calendar_grid, "source", SYNC_CREATE);
+
+        calchooser.notify["current-source"].connect ((s, p) => {
+            calendar_button.tooltip_text = "%s—%s".printf (current_calendar_grid.label, current_calendar_grid.location);
```

Comment:
The tooltip text does not get set unless current-source changes? This occurs also in `master` though.

Reply:
Yeah I noticed that too. I think we'll revisit this anyways when porting to Gtk.DropDown, but like you said already occurs in master so wasn't too invested in try to fix it in this branch